### DISCUSSION TITLE
Pnp Search Result web part picture resolution

### DISCUSSION THREAD

**Discussion by davidfolescu** [microsoft-search/pnp-modern-search#4065] [Q&A]
Hello,

App version: 4.11.1.0 

We have case where we are using the modern Pnp web parts and we would like to enlarge the pictures for the top news. We managed change the width but we are losing the sharpness of the pictures. The Pnp Search Result is using the "AutoPreviewImageUrl” property which offers us a picture of 400 pixels, there for the blurriness of the pictures. Is the a possibility to increase the number of pixels to 960 like in the Highlighted content web part ( standard SharePoint Web Part) 
See screenshot below as an example between Pnp Search Result and Highlighted content.
![image](https://github.com/user-attachments/assets/eb76908e-755b-4bb4-b108-b856da73bdb8)

**Comment by kasperbolarsen** [comment]
Is the URLAnchor property any better?

**Reply by davidfolescu** [reply]
Hello,
Can you provide us more details about the URLAnchor property?

**Reply by kasperbolarsen** [reply]
Not really, I could just see it exists and seems to provide a higher res picture

**Reply by davidfolescu** [reply]
Thank you , we tried , the URLAnchor property and we are getting a picture of 300x200 picture, which is smaller than the picture returned  by the AutoPreviewImageUrl property which is 400,
also i have to mention that when adding the URLAnchor property as slot field in the slot name PreviewImageUrl2 ( the normal one is attributed to AutoPreviewImageUrl) no picture is shown.
is there any way to manipulate the URLAnchor property so we can achieve our goal?

**Reply by kasperbolarsen** [reply]
You could use one of the debugging tools here,https://microsoft-search.github.io/pnp-modern-search/using-query-tools-to-verify-issues/
To inspect the available properties returned by the API. Perhaps one of them might give you an idea?
